verification/curator-service/ui/src/components/VerificationStatusHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import { IconButton, Tooltip } from '@material-ui/core';
+
+import React from 'react';
+import { ReactComponent as VerificationHeaderSvg } from './assets/verification_header.svg';
+
+export default function VerificationStatusHeader(): JSX.Element {
+    return (
+        <Tooltip title="Coming soon: toggle verification status">
```

Comment:
What do you mean by toggle? Usually clicking on a header sorts the column (which we currently have disabled on the table)

Reply:
That's how I read [the description in the mocks](https://www.figma.com/file/9HJlRkTBCMEQdkJ6WQJYkv/061120_Global.Health-Line?node-id=981%3A901). Looking back, I could have taken that to mean the header is used to filter, as well. I'll change this text in the meantime.